### ISSUE TITLE
Certain KaTeX does not render in exported PDF

### ISSUE THREAD

**Issue by eh8** [laurent22/joplin#2415]
![image](https://user-images.githubusercontent.com/22990581/73491248-2db40680-437c-11ea-9795-24db76da55dd.png)

![image](https://user-images.githubusercontent.com/22990581/73491266-37d60500-437c-11ea-813b-1386c4d27e55.png)

## Environment

Joplin version: 1.0.179
Platform: Linux
OS specifcs: Arch

## Steps To Reproduce

1. Use notation like `\R, \N, \Z`
2. Export PDF and observe such symbols disappear

This seems to happen with and without any `userstyle.css` content.

For a more realistic reproduction, try exporting this:
```
$(\forall x \in \N)(\forall y \in \Z \, | \, (x + y < 0) \lor x - y \geq 0)$

$(x \in \R)\land(y \in \R)\land(x+y=5)$

$(\exist x \in \R)(\exist y \in \R)(x + y = 5 \land 3x-y = 7)$

$(\forall x \in \R)(\forall y \in \R)(y \leq x \lor (\exist r > 0)(x + r > y))$

$(\exist x \in \R)(\forall y \in \R)(xy \neq x)$
```

For a complete list of unsupported characters when exporting, view the KaTeX documentation:

https://katex.org/docs/supported.html

Describe what you expected to happen:

The rendered markdown from first image should match the PDF

![image](https://user-images.githubusercontent.com/22990581/73491509-cba7d100-437c-11ea-808a-c298b96261bc.png)

Looks like there is some missing LaTeX font files.

## Logfile

[log.txt](https://github.com/laurent22/joplin/files/4136321/log.txt)

**Comment by eh8** [comment]
I added 

1. `KaTeX_AMS-Regular.ttf`
2. `KaTeX_AMS-Regular.woff`
3. `KaTeX_AMS-Regular.woff2`

to `usr/share/joplin/resources/app/gui/note-viewer/pluginAssets/katex/fonts/` but this did not solve the issue. However, now I have proper AMS-rendered symbols :)

**Workaround:** Disable all `userstyle.css` code

**Comment by laurent22** [comment]
I've just tried your Katex code and it looks like it's all been fixed in 95e0e8d459a55cc81dcd7c617cf6619f2c304696 If you notice anything else not working, please let me know.

**Comment by eh8** [comment]
Awesome, thanks so much! This is brilliant.
